modules/core/src/v2/coins/xlm.ts

```diff
@@ -398,7 +401,7 @@ export class Xlm extends BaseCoin {
    * @return true if stellar username is valid
    */
   isValidStellarUsername(username: string): boolean {
-    return /^[a-z0-9\-\_\.\+\@]+$/.test(username);
+    return /^[a-z0-9\-_.+@]+$/.test(username);
```

Comment:
no need to escape + and @?

Reply:
Nope, since it's inside a character class (brackets).

Reply:
```
$ node
> /[@.+]/.test('@')
true
```